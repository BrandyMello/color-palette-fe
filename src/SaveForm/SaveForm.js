import React from 'react';

const SaveForm = () => {
  return (
    <div>
      Hey
    </div>
  )
}

export default SaveForm