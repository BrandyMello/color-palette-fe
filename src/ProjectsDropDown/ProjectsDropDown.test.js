import React from 'react';
import { shallow } from 'enzyme';
import SaveForm from './ProjectsDropDown';

describe('SaveForm', () => {
  it('should return true', () => {
    expect(true).toEqual(true)
  })
})