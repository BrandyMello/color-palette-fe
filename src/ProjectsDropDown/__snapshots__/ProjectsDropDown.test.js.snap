// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsDropDown should match snapshot 1`] = `
<form
  className="newPalette"
>
  <select
    className="new_palette_name"
    onChange={[Function]}
  >
    <option
      key="0"
      value="Warm Colors"
    >
      Warm Colors
    </option>
    <option
      key="1"
      value="Cool Colors"
    >
      Cool Colors
    </option>
  </select>
</form>
`;
